vaadin-rich-text-editor-flow-parent/vaadin-rich-text-editor-flow-integration-tests/src/test/java/com/vaadin/flow/component/richtexteditor/tests/RichTextEditorValueIT.java

```diff
@@ -0,0 +1,211 @@
+package com.vaadin.flow.component.richtexteditor.tests;
+
+import com.vaadin.flow.component.richtexteditor.testbench.RichTextEditorElement;
+import com.vaadin.flow.testutil.TestPath;
+import com.vaadin.testbench.TestBenchElement;
+import com.vaadin.tests.AbstractComponentIT;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+@TestPath("vaadin-rich-text-editor/value")
+public class RichTextEditorValueIT extends AbstractComponentIT {
```

Comment:
These ITs have value as there is some JS logic involved in synchronizing HTML value -> Delta value. In the upcoming refactoring, the sync will also work the other way around, which is why I added tests for all scenarios right away.

Note that some assertions and tests are disabled, as the current sync logic, as well as tracking the previous HTML value, is partially broken. I will enable those in the upcoming refactoring.